f/frizzle/comapeo/comapeo_observations.py

```diff
@@ -1,16 +1,18 @@
 # requirements:
 # psycopg2-binary
 # requests~=2.32
+# retrying~=1.3
 
 import logging
+import mimetypes
 import os
 import re
 
 import psycopg2
-import requests
 from psycopg2 import errors
+import requests
```

Comment:
Was moving this import done automatically and if so by what?

The setup described in https://github.com/ConservationMetrics/gc-scripts-hub/pull/8 (specifically `"source.organizeImports": "explicit"`) would have kept this as it previously was, not made this change.  If Windmill's UI changed this then that's an interesting clash.

Reply:
No, I moved it because I thought it made more sense to follow `psycopg2` related imports.